server/server.js

```diff
@@ -8,6 +8,9 @@ var app = express();
 app.use(bodyParser.json());
 app.use(express.static(__dirname + '/../public'));
 
+var bocksSchema = require('./schemaModel.js');
+mongoose.connect(__dirname, 'mongodb://host/db');
```

Comment:
@drewkosta (testing code comments) is this line working for you? it's not for me. thanks.